src/sentry/integrations/gitlab/integration.py

```diff
@@ -212,13 +213,14 @@ def build_integration(self, state):
         group = self.get_group_info(data['access_token'], state['installation_data'])
         scopes = sorted(GitlabIdentityProvider.oauth_scopes)
         base_url = state['installation_data']['url']
+        domain_name = '%s/%s' % (base_url.replace('https://', ''), group['path'])
```

Comment:
Note changed the `domain_name` from `example.gitlab.com/groups/my-group` to `example.gitlab.com/my-group`

Reply:
Do folks ever run gitlab without SSL? Should we also be replacing `http://`?

Reply:
Mark I honestly don't know. But It's a quick change to add something that will handle that regardless.